deploy/common/k8sbased.sh

```diff
@@ -248,8 +248,10 @@ function launch_central {
       add_args "--enable-telemetry=false"
     fi
 
-    if [[ -n "${ROX_OPENSHIFT_VERSION}" ]]; then
-      add_args "--openshift-version=${ROX_OPENSHIFT_VERSION}"
+    if [[ "${ORCH}" == "openshift" ]]; then
+      if [[ -n "${ROX_OPENSHIFT_VERSION}" ]]; then
+        add_args "--openshift-version=${ROX_OPENSHIFT_VERSION}"
+      fi
```

Comment:
Does this imply that there are cases where `ROX_OPENSHIFT_VERSION` is set, but orchestrator is not openshift? Perhaps we should fix that instead? 🤔

Reply:
Well, a quick scan didn't uncover such cases. I just figured that from the perspective of `k8sbased.sh` it would make more sense to only attempt openshift configuration if we are running on openshift.